README.md

```diff
@@ -41,7 +41,7 @@ The following stable releases are supported. The development branch usually work
 ## Test status of cloud providers
 
 * [Betacloud](https://www.betacloud.de): Works
-* [Citycloud](https://www.citycloud.com): Works (need to change disk names from sdX to vdX, pass ``drives_vdx: true`` in environment). You can use ``environment-CityCloud.yml`` to get a set of working parameters.
+* [Citycloud](https://www.citycloud.com): Works
```

Comment:
The pointer to environment-CityCloud.yml should be left in (it also specifies flavors, image names, az names that work on CC).